### PR TITLE
[CI] Fix Doctor-RST's code blocks order warnings

### PR DESCRIPTION
**Pull request by alexandre-daubois**

### CODE REVIEW

**Comment by OskarStark** [comment]
Oh not sure about the order yet.

@javiereguiluz do we want to have stand-alone as first tab or Symfony usage?

**Comment by wouterj** [comment]
The documentation is focussed on the framework user, so I would say php-standalone should always be last.

**Comment by alexandre-daubois** [comment]
Oh alright, didn't think it was a question not answered yet 🙂 I agree with Wouter. But if the order stays the same as it is now, the PR is ready. Otherwise, I'll close it!

**Comment by OskarStark** [comment]
I think we should make a difference:

If we only have `php-standalone` and `php-symfony` in a configuration-block, the order should be `php-symfony`, `php-standalone`, but if we have more in a configuration-block like:

  792: Please use the following order for your code blocks: "php-standalone, yaml, xml, php"

we should move `php-standalone` to the end after `php`. WDYT @wouterj or are you fine with the proposed order (`php-standalone, yaml, xml, php`), if we have more code-blocks in the configuration-block?

If you agree, we need to adjust the rule in DOCtor

cc @alamirault

**Comment by javiereguiluz** [comment]
I like your last proposal Oskar

**Comment by wouterj** [comment]
Yes, I would say independent of the other formats `php-standalone` should always be last.

`php-symfony`, `php-standalone`
`yaml`, `xml,` `php`, `php-standalone`
`xliff`, `yaml`, `php`, `php-standalone`
etc.

**Comment by OskarStark** [comment]
I will be on vacation next week, so I propose to merge this PR as is, to have a stable build and after I am back, I will take care of the correct order.

Or @alamirault wants to do another PR against DOCtor 😄

File: components/form.rst [hunk 1]
```diff
@@ -533,19 +533,6 @@ by :method:`Symfony\\Component\\Form\\Form::handleRequest` to determine whether
 
 .. configuration-block::
 
-    .. code-block:: php-standalone
```
Comment (xabbuh) [inline components/form.rst:536]: Was this example removed intentionally?

**Comment by xabbuh** [comment]
> I will be on vacation next week, so I propose to merge this PR as is, to have a stable build and after I am back, I will take care of the correct order.

Sounds like a solid plan to me. I have one question though (see above).

File: components/form.rst (hunk 1 above)
Comment (OskarStark) [reply components/form.rst:536]: The diff looks weird, maybe use Split View, but from what I can see, it still there 🤔

Comment (alexandre-daubois) [reply components/form.rst:536]: Yep, Oskar's right! Can't tell why the diff looks like this but I just moved the code snippet 🙂

**Comment by wouterj** [comment]
I don't think we should favor worse online docs over a red CI for a week.

**Comment by OskarStark** [comment]
Ok

**Comment by OskarStark** [comment]
I merged
* https://github.com/OskarStark/doctor-rst/pull/1272
and released: https://github.com/OskarStark/doctor-rst/releases/tag/1.35.1

**Comment by alexandre-daubois** [comment]
Thank you Oskar, I'm closing this one! :slightly_smiling_face:

**Pull request closed without merging**